wazo_call_logd/cel_interpretor.py

```diff
@@ -252,6 +254,38 @@ def interpret_wazo_meeting_name(self, cel, call):
 
         return call
 
+    def interpret_wazo_user_missed_call(self, cel, call):
+        extra = extract_cel_extra(cel.extra)
+        if not extra:
+            return
+
+        extra_tokens = extra['extra'].split(',')
+        wazo_tenant_uuid = extra_tokens[0].split(': ')[1]
+        source_user_uuid = extra_tokens[1].split(': ')[1]
+        destination_user_uuid = extra_tokens[2].split(': ')[1]
+        destination_exten = extra_tokens[3].split(': ')[1]
+        source_name = extra_tokens[4].split(': ')[1]
+        destination_name = extra_tokens[5].split(': ')[1]
+
+        source_participant = CallLogParticipant(
+            role='source',
+            user_uuid=source_user_uuid,
+        )
+        call.participants.append(source_participant)
+        destination_participant = CallLogParticipant(
```

Comment:
`tags` should be added to the participants

Reply:
I have added the tags field to both participants, but what should it be set to? I know it is a list, but for now I set it to be `['Montreal', 'Quebec']` for `source_participant`, and `['Paris', 'France']` for `destination_participant`. I am not sure though if it is correct...

Reply:
The tags come from the wazo-confd user configuration. You can see what it should be [here](https://github.com/wazo-platform/wazo-call-logd/blob/master/wazo_call_logd/generator.py#L91-L108)